fontbe/src/gvar.rs

```diff
@@ -110,9 +107,8 @@ mod tests {
                 v if v == glyph_without_var => Vec::new(),
                 // At the maximum extent (normalized pos 1.0) of our axis, add +1, +1
                 v if v == glyph_with_var => vec![GlyphDeltas::new(
-                    Tuple::new(vec![F2Dot14::from_f32(1.0)]),
+                    vec![gvar::Tent::new(F2Dot14::from_f32(1.0), None)],
```

Comment:
i'm curious why not just import `Tent` like we already do for GlyphDelta and GlyphDeltas?

Reply:
oh I see now, because fontir also defines a struct named Tent. (Oh look, that one also has fileds ordered as {min, peak, max}, not {peak, min, max}, but nevermind).